### ISSUE TITLE
[Linux] "Select an alternative EXE to run" file chooser icon does nothing when clicked

### ISSUE THREAD

**Issue by Nocifer** [Heroic-Games-Launcher/HeroicGamesLauncher#1644]
### Describe the bug

When trying to set an alternative EXE for a game, clicking the "+" button in the text box does nothing, i.e. no file chooser is launched.

### Add logs

No error is printed, it just doesn't do anything.

### Steps to reproduce

1. Go to Game Settings -> Other
2. Click on the "+" button inside the "Select an alternative EXE to run" text box.
3. Nothing happens.

### Expected behavior

A file chooser should be launched in order for the user to find and select the EXE of their choice.

### System Information

- OS: Arch Linux
- Heroic Version: 2.4.0-beta.2

**Comment by flavioislima** [comment]
Working fine for me:
![image](https://user-images.githubusercontent.com/26871415/181478337-b55d75ed-c161-4211-9770-d9e453c112d9.png)

Can you give more details on your system like which DE or WM are you using?
How did you install Heroic?

**Comment by Nocifer** [comment]
I'm using KDE with no significant tweaks; and I installed Heroic using `heroic-2.4.0-beta.2.pacman`.

**Comment by Nocifer** [comment]
Tested both 2.3.10 and 2.4.0-beta.2 with a fresh install, including a freshly installed (not imported) GOG game and a newly created wine prefix. This is broken in both 2.3.10 2.4.0-beta.2, at least for me. It turns out that there is an error printed in the debug console:

```index.tsx:201 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'install')
    at index.tsx:201:33
    at f (regeneratorRuntime.js:86:17)
    at Generator._invoke (regeneratorRuntime.js:66:24)
    at Generator.next (regeneratorRuntime.js:117:21)
    at r (asyncToGenerator.js:3:20)
    at s (asyncToGenerator.js:25:9)
```

It's practically the same error as reported in #1643.

**Comment by arielj** [comment]
Can you test older Heroic versions? we call Electron's native file picker, maybe something changed in later versions of electron? maybe an older version of heroic with an older version of electron works.

At least to know when the problem appeared, I can't reproduce this either en Linux Mint and CInnamon.

**Comment by Nocifer** [comment]
For what it's worth, I just tested this in an out-of-the-box OpenSuse VM with the Flatpak version of Heroic 2.3.10 and it doesn't work there either, with the same error printed in the debug console:

```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'install')
    at index.tsx:139:33
    at f (regeneratorRuntime.js:86:17)
    at Generator._invoke (regeneratorRuntime.js:66:24)
    at Generator.next (regeneratorRuntime.js:117:21)
    at r (asyncToGenerator.js:3:20)
    at s (asyncToGenerator.js:25:9)
```

So I guess it's not some issue with my system or my configuration after all.

I'll try to test some more in the coming days, but we're suffering a heatwave in my country and I'm not sure if I'll be able to force myself to spend my time debugging instead of going to the beach 24/7 :P

**Comment by Nocccer** [comment]
Just try some common distro. Ubuntu, Pop_OS, manjaro, arch, fedora.

**Comment by arielj** [comment]
I can confirm this happens ONLY with GOG games. I can reproduce this error both in the `Select an alternative EXE to run` and clicking the `Run EXE in prefix` options, but only for GOG games, Epic games open those options just fine.

I'm seeing that same error in the dev tools console.

**Comment by flavioislima** [comment]
I think I know the issue. will send a hotfix

**Comment by flavioislima** [comment]
Actually, this was fixed on my PR #1646 since now we have the runner in the routes.
This happened because the GOG games were not getting the gameInfo correctly.

@Nocifer can you test with the AppImage from my PR to confirm?

**Comment by arielj** [comment]
I can confirm this works fine for me now in the beta branch, clicking `Select an alternative EXE to run` for a GOG game opened the file selector. `Run EXE on Prefix` also worked fine.
